recipes/dagger-io/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dagger-io" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dagger_io-{{ version }}.tar.gz
+  sha256: 386c14beaab5e83d93328e8e74f78add67e589b2523ecf9707ae6f8d17caa095
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - dagger-py = dagger.cli:app
+
+requirements:
+  host:
+    - python >=3.10,<4.0
+    - pip
+    - poetry
+  run:
+    - python >=3.10,<4.0
+    - anyio >=3.6.2
+    - attrs >=22.1.0
+    - cattrs >=22.2.0
+    - gql >=3.4.0
+    - httpx >=0.23.1
+    - typer >=0.6.1
+    - beartype >=0.11.0
+    - platformdirs >=2.6.2
+
+test:
+  imports:
+    - dagger
+  commands:
+    - pip check
```

Comment:
Maybe it would be worthwhile to run `dagger --help` to ensure that the CLI is functioning?

Reply:
The CLI is only being used internally (from dagger's CI). People won't be using it for now, it's not advertised or documented. I've considered removing that entrypoint so it doesn't create the executable, but that's for another release.

If I add the command, how does it work in windows? It would `dagger-py --help`, but in windows it's `dagger.py.exe`.

Reply:
I realize you meant the [dagger CLI](https://github.com/dagger/dagger/releases/tag/v0.3.9), but that's not included when you install `dagger-io`. It's fetched automatically on-demand during first execution globally.

Reply:
Oh! I had no idea. Yeah, it was just a small suggestion.